swifter/swifter.py

```diff
@@ -1,13 +1,16 @@
-import pandas as pd
-from psutil import cpu_count
-from dask import dataframe as dd
 import timeit
 import warnings
+import pandas as pd
+
+from psutil import cpu_count
 from tqdm import tqdm
-from .tqdm_dask_progressbar import TQDMDaskProgressBar
+from dask import dataframe as dd
+
+import tqdm_dask_progressbar
```

Comment:
Pretty sure it needs the .tqdm_dask_progressbar for a relative import when launched as a python package on pip